CHANGELOG.md

```diff
@@ -12,15 +12,19 @@ and this project adheres to [Python Versioning](http://legacy.python.org/dev/pep
 * #### NI-DMM
   * #### Added
   * #### Changed
+    * Added default values for some methods.
   * #### Removed
+    * Removed some unnecessary methods.
```

Comment:
Removed methods that aren’t useful in the Python bindings.

Reply:
Fixed